test/harness/ConstantProduct.ts

```diff
@@ -284,7 +286,13 @@ export async function burnLiquidity(poolIndex, amount, withdrawType, unwrapBento
       },
     ];
     const burnData = utils.defaultAbiCoder.encode(["address", "bool"], [accounts[0].address, unwrapBento]);
-    await router.burnLiquidity(pool.address, amount, burnData, minWithdrawals);
+    burnLiquidityPromise = router.burnLiquidity(pool.address, amount, burnData, minWithdrawals);
+  }
+
+  if (token0.address < token1.address) {
```

Comment:
This should always be true, if you encounter a case where it is false it would mean there is a bug in the factory

Reply:
Ah, or are the token0 and token1 arbitrary here?

Reply:
I guess you are right, this if else is not necessary, I wrote a similar if else in other test classes but removed them eventually. I missed to delete this one. Will do in future pull request if possible. Thanks for merging !